packages/js/examples/react-audio-widgets/.gitignore

```diff
@@ -7,4 +7,5 @@ coverage
 cypress/integration/examples
 cypress/videos/*
 cypress/screenshots/*
-.env
\ No newline at end of file
+.env
+yarn.lock
```

Comment:
Why add the lockfile to gitignore? Also, it looks like the file was committed to this PR

Reply:
When we use the command `npm ci` we need to remove the yarn.lock, because the GITHUB/ACTIONS complain about that.

Reply:
```
Run cypress-io/github-action@v2
/usr/bin/yarn --frozen-lockfile
yarn install v1.22.5
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
warning cypress > @cypress/request > har-validator@5.1.5: this library is no longer supported
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: The process '/usr/bin/yarn' failed with exit code 1
`
``

Reply:
I think they meant you should remove package-lock.json file instead, as it could potentially be used instead of yarn.lock.